.github/workflows/pr-cypress.yml

```diff
@@ -112,3 +100,4 @@ jobs:
           # Passing the GitHub token lets this action correctly
           # determine the unique run id necessary to re-run the checks
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # DEBUG: '@cypress/github-action'
```

Comment:
What is this used for?

Reply:
I'm going to remove it but I left it here to remind me to ask if we wanted to use any inputs on GitHub actions to turn debugging on/off.